tests/cypress/page-object/dateTimePicker.ts

```diff
@@ -0,0 +1,75 @@
+import {DatePicker} from './datePicker';
+
+export class DateTimePicker extends DatePicker {
+    protected getNowTime(separator: string): string {
+        const date = new Date();
+        const hours = date.getHours();
+        const minutes = date.getMinutes();
+
+        const stringHour = hours < 10 ? `0${hours}` : hours;
+        const stringMinutes = minutes < 10 ? `0${minutes}` : minutes;
+
+        return `${stringHour}${separator}${stringMinutes}`;
+    }
+
+    protected nearestHalfHour(separator: string) {
+        const nowTime = this.getNowTime(separator);
+        const hourParts = nowTime.split(separator);
+        const hourPartsInt = [];
+        for (let i = 0; i < hourParts.length; i++) {
+            hourPartsInt[i] = parseInt(hourParts[i], 10);
+        }
+
+        if (hourPartsInt[1] >= 45) {
+            hourPartsInt[0]++;
+        }
+
+        if (hourPartsInt[1] <= 15 || hourPartsInt[1] >= 45) {
+            hourPartsInt[1] = 0;
+        } else {
+            hourPartsInt[1] = 30;
+        }
+
+        const stringHour = hourPartsInt[0] < 10 ? `0${hourPartsInt[0]}` : hourPartsInt[0];
+        const stringMinutes = hourPartsInt[1] < 10 ? `0${hourPartsInt[1]}` : hourPartsInt[1];
+
+        return `${stringHour}${separator}${stringMinutes}`;
+    }
+
+    public getTimePicker(): Cypress.Chainable {
+        return cy.get('input[id="qant:pickers_datetimepicker"]');
```

Comment:
`qant:pickers_datetimepicker` should be set as parameter, otherwise we will not be able to use this function for other node types

Reply:
As the id is used in several function of this class, you can set the value of the id as a global property that you set when you call the constructor of this object